R/model_WRMF.R

```diff
@@ -88,7 +88,7 @@ WRMF = R6::R6Class(
       private$with_bias = with_bias
 
       solver_codes = c("cholesky", "conjugate_gradient", "nnls")
-      private$solver_code = match(solver, solver_codes) - 1L
+      private$solver_code = match(solver[1], solver_codes) - 1L
```

Comment:
I think `[1]` is redundant as `match.arg()` above already makes sure that `solver` is a vector of length one.